assets/js/util/integrationInfo.js

```diff
@@ -43,6 +43,13 @@ export const PREMADE_CHANNEL_TYPES = [
     info: "Cargo is an in-house mapping tool used at Helium.",
     docLink: "https://docs.helium.com/use-the-network/console/integrations/cargo"
   },
+  {
+    name: "Google Sheets",
+    link: "/integrations/new/googlesheet",
+    img: `${Google}` ,
+    info: "This Integration streamlines sending data to a Google Sheet by using a Google Form.",
+    docLink: "https://docs.helium.com/use-the-network/console/integrations/tago"
```

Comment:
can we use the correct placeholder so we don't gotta change this later

Reply:
yisss